README.md

```diff
@@ -24,6 +24,11 @@ Fingerprint Pro CloudFront Integration is responsible for
 
 This [improves](https://dev.fingerprint.com/docs/cloudfront-proxy-integration#the-benefits-of-using-the-cloudfront-integration) both accuracy and reliability of visitor identification and bot detection on your site.
 
+> [!IMPORTANT]  
+> Cloudfront integration v1 has been [deprecated](https://dev.fingerprint.com/docs/cloudfront-proxy-integration).
```

Comment:
Could you change "Cloudfront" to "CloudFront" with capital F please? @JuroUhlar

Reply:
Fixed, thank you! good eyes!